libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -504,6 +507,9 @@ class AP_InertialSensor : AP_AccelCal_Client
     // are we in HIL mode?
     bool _hil_mode:1;
 
+    // are we in SITL mode?
+    bool _sitl_mode:1;
```

Comment:
like what @tridge said, better to use #if CONFIG_HAL_BOARD == HAL_BOARD_SITL

Reply:
Will revert out the entire change to AP_InertialSensor.